### PR TITLE
Add general overview for the website

### LINKED ISSUE #2: Decide on general organization for the website
We need to decide on a general organization for the website. Here is a suggestion (feedback needed):
- Introduction
  - if people are seeking to also implement shimming hardware, they can check out the shim amplifier design included elsewhere on the website. --> LINK TO "HARDWARE"
- Installation
- Getting started
- Support
- Contributing
  - General
  - API documentation
- Other ressources
  - Hardware
    - Description of hardware with amplifiers
    - Assembly
- Contributors
- License

### PR DESCRIPTION
**Pull request by po09i**
The purpose of this PR is to implement the overall overview of the website. Fixes #2

### CODE REVIEW

**Comment by po09i** [comment]
Some sections like the introduction, installation, getting started, help, etc are subsections of the README inside the shimming-toolbox repo. Should we consider splitting them up in their own separate files inside shimming-toolbox or make a script that will split them up when deploying on shimming-toolbox.org repo?

**Comment by po09i** [comment]
I used the shimming-toolbox README to fill out the seperate sections of the website. The general overview is there along with the documentation inside contributing/api-documentation.

Following PRs could fill the missing information, notably :
- Support
- Hardware Description
- Hardware Assembly
- Contributors

As well as polishing 
- Introduction
- Getting started

File: docs/contributing/contributing.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+# Code Review
+
+[What is code review?](https://help.github.com/articles/about-pull-request-reviews/)
+
+Any changes submitted to the master branch will go through code review. For a pull request to be accepted it must have:
+
+* At least 2 members approve the changes.
+* TravisCI must pass successfully
+
+Reviewing members are :
```
Comment (jcohenadad) [inline docs/contributing/contributing.md:143]:
```suggestion
Reviewing members are:
```

File: docs/contributing/contributing.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+To help prioritize the request, add labels that describe the type and impact of the change. A change can have multiple types if it is appropriate but would have only 1 impact label. Such as `bug documentation fix:patch`.
+See [label definitions](https://github.com/neuropoly/realtime_shimming/labels) on how to categorize the issue and pull request.
+
+TODO: ADD AN EXAMPLE WITH LINK TO PR
+
+# Code Review
+
+[What is code review?](https://help.github.com/articles/about-pull-request-reviews/)
+
+Any changes submitted to the master branch will go through code review. For a pull request to be accepted it must have:
```
Comment (jcohenadad) [inline docs/contributing/contributing.md:138]:
```suggestion
Any changes submitted to the master branch will go through code review. For a pull request to be accepted:
```

File: docs/contributing/contributing.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+* At least 2 members approve the changes.
```
Comment (jcohenadad) [inline docs/contributing/contributing.md:140]:
```suggestion
* At least 1 member should approve the changes.
```

File: docs/contributing/contributing.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+* TravisCI must pass successfully
+
+Reviewing members are :
+* @jcohenadad
+* @rtopfer
```
Comment (jcohenadad) [inline docs/contributing/contributing.md:145]:
```suggestion
* @jcohenadad
* @rtopfer
* @po09i 
```

File: docs/contributors.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# Contributors
+
+[List of contributors.](https://github.com/neuropoly/realtime_shimming/graphs/contributors)
```
Comment (jcohenadad) [inline docs/contributors.md:3]:
```suggestion
[List of contributors.](https://github.com/shimming-toolbox/shimming-toolbox/graphs/contributors)
```

File: docs/getting_started.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+TODO: Procedure for creating calibration maps. (Also see: Topfer et al. MRM 2016)
+
+Once this experimental gradient-echo data is available, one need needs to provide the paths to the DICOM images and to define the corresponding experimental currents used in ShimOpt_MyNewShim.declarecalibrationparameters()...
+
+TODO cont...
+
+List of available maps:
+- UNF: https://drive.google.com/drive/folders/1DSCTeh9qRCgS55fWLGtsNeYdq8DsRyMg
+
+Download map and copy it under $PATH_REALTIMESHIMMING/data/
```
Comment (jcohenadad) [inline docs/getting_started.md:31]: link is dead, and is this procedure still valid?

File: docs/getting_started.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+- Control room
+   - Connect sensor probe to station
+  - Plug the optic-to-serial-to-usb (PICTURE) to the computer.
+    - If you are running macOS, you might need to install [this driver](http://www.prolific.com.tw/US/ShowProduct.aspx?p_id=229&pcid=41).
+  - In the terminal, change to the home directory and open matlab
+  - Copy the file `shim_params_template.m` to `shim_params.m` and edit `shim_params.m` according to your setup.
+  - Launch experiment by instantiating a ShimUse object with the desired parameters: Shims = ShimUse( shim_params )
+
+- See example: realtime_shimming/example/XXX
```
Comment (jcohenadad) [inline docs/getting_started.md:50]: for all the things that are left unfinished (i.e. TODO), open an issue.

File: docs/getting_started.md (hunk 6 above)
Comment (po09i) [reply docs/getting_started.md:31]: @rtopfer Do you have a new link and is it still valid?

**Comment by po09i** [comment]
All the files outside the API-doc were taken from the Shimming-Toolbox README and CONTRIBUTING files. We should change things there as well.

File: docs/getting_started.md (hunk 6 above)
Comment (jcohenadad) [reply docs/getting_started.md:31]: gdrive is not sustainable-- we should move to osf as we've started to do-- @po09i can you please initiate? thx (open issue so we track down progress)

**Comment by jcohenadad** [comment]
> All the files outside the API-doc were taken from the Shimming-Toolbox README and CONTRIBUTING files. We should change things there as well.

another pros for merging everything into a single repos 😉

File: docs/getting_started.md (hunk 6 above)
Comment (po09i) [reply docs/getting_started.md:31]: Opened issue #21

**Pull request merged**